backend/src/utils/userUtils.ts

```diff
@@ -29,6 +29,9 @@ export type OpenShiftUser = {
     name: string;
     uid: string;
     resourceVersion: string;
+    annotations: {
```

Comment:
Annotations are optional.
```suggestion
    annotations?: {
```